src/components/authors-list.tsx

```diff
@@ -0,0 +1,37 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import Author from './author';
+
+const AuthorsList = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  margin: 5rem 0;
+  align-items: center;
+  color: var(--gray7);
+  text-transform: uppercase;
+
+  span {
+    margin: 0.5rem;
+  }
+`;
+
+type Props = {
+  authors: string[];
+};
+
+export default ({ authors }: Props) => {
+  if (!authors) {
+    return null;
+  }
+
+  return (
+    <AuthorsList>
+      <span>Contributors</span>
+      {authors &&
+        authors.map(
```

Comment:
Perhaps this could use array check and map?

```
{Array.isArray(authors) && authors.map(
  ...
)}
```

Reply:
Both translate to the same thing I suppose a `truthy`check is normally more performant. It's postmatter also there's a TypeScript check that makes sure this is an array with string values `string[]`. Better than `isArray`.